mlos_bench/mlos_bench/tests/config/schemas/environments/test-cases/good/full/local_env-full.jsonc

```diff
@@ -25,7 +25,18 @@
         "teardown": [
             "/bin/bash -c true"
         ],
+
         "read_results_file": "/tmp/results.json",
-        "dump_params_file": "/tmp/dump_params.json"
+        "dump_params_file": "/tmp/dump_params.json",
+
+        "shell_params_match": [
+            "^MLOS_[a-zA-Z0-9_]+$"
+        ],
+        "shell_params": [
+            "foo"
+        ],
+        "shell_params_mapping": {
+            "gets_mapped_to_this_env_name": "/source/param/name/that/is/otherwise/invalid/env/name"
```

Comment:
Note: the mapping is maybe reverse from the expected `"/source/param": "target_env_name"` so that we can enforce uniqueness of `"target_env_name"`